MerlinAU.sh

```diff
@@ -146,10 +146,27 @@ readonly IPv4octet_RegEx="([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])"
 readonly IPv4addrs_RegEx="(${IPv4octet_RegEx}\.){3}${IPv4octet_RegEx}"
 readonly IPv4privt_RegEx="(^10\.|^172\.1[6-9]\.|^172\.2[0-9]\.|^172\.3[0-1]\.|^192\.168\.)"
 
+##----------------------------------------##
+## Modified by Martinski W. [2024-Oct-03] ##
+##----------------------------------------##
 readonly fwInstalledBaseVers="$(nvram get firmver | sed 's/\.//g')"
 readonly fwInstalledBuildVers="$(nvram get buildno)"
 readonly fwInstalledExtendNum="$(nvram get extendno)"
 readonly fwInstalledInnerVers="$(nvram get innerver)"
+readonly fwInstalledBranchVer="${fwInstalledBaseVers}.$(echo "$fwInstalledBuildVers" | awk -F'.' '{print $1}')"
+
+##-------------------------------------##
+## Added by Martinski W. [2024-Oct-03] ##
+##-------------------------------------##
+readonly MinSupportedFW_3004_386_Ver="3004.386.12.6"
+readonly MinSupportedFW_3004_388_Ver="3004.388.6.2"
+readonly MinSupportedFW_3006_102_Ver="3004.388.8.0"
```

Comment:
Ah yes, I see what you did here to allow some routers to upgrade from 3004 to 3006